### DISCUSSION TITLE
Return of money

### DISCUSSION THREAD

**Discussion by Viljur** [community/community#102874] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is there any way to get a refund money if I mistakenly bought 1 more account with lama?

**Comment by KowyXYZ** [comment]
All sales are final. No refunds. 
https://lamaretail.com/pages/return-exchange-policy

**Comment by queenofcorgis** [comment]
Hi @Viljur ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Lama. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
